app/assets/javascripts/discourse/app/controllers/user-invited-show.js

```diff
@@ -31,10 +31,16 @@ export default Controller.extend({
       this.user,
       this.filter,
       this.searchTerm
-    ).then(invites => this.set("model", invites));
+    ).then((invites) => this.set("model", invites));
   }, INPUT_DELAY),
 
   inviteRedeemed: equal("filter", "redeemed"),
+  invitePending: equal("filter", "pending"),
+
+  @discourseComputed("filter")
+  inviteLinks(filter) {
+    return filter === "links" && this.currentUser.get("admin");
```

Comment:
maybe we should have a constant for "links" here

Reply:
It's not being used anywhere else... also we use the same pattern for checking for "pending" and "redeemed" tabs, so I am not sure about this.